src/Menu.js

```diff
@@ -1,14 +1,15 @@
 class Menu {
   constructor() {
-    
+    this.recipesToCook = [];
   }
-}
-
-
-
-
 
+  addToMenu(recipe) {
+    if (!this.recipesToCook.includes(recipe)) {
```

Comment:
Looks like that'll do. By the way, this is a great way for sad path testing for duplicates